### PR TITLE
Improve e2e tests

### PR DESCRIPTION
**Pull request by yodigos**

### CODE REVIEW

File: frontend/webapp/cypress.config.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { defineConfig } from "cypress";
+
+export default defineConfig({
+  projectId: 'aydhz4',
```
Comment (alonkeyval) [inline frontend/webapp/cypress.config.ts:4]: is this something that can be exposed?

File: frontend/webapp/cypress/fixtures/example.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (alonkeyval) [inline frontend/webapp/cypress/fixtures/example.json:1]: we need this file?

File: tests/common/ui-tests/run_cypress_tests.sh [hunk 3]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+echo "Running Cypress tests"
+cd ../../../frontend/webapp || exit
+npx cypress run --record --key d355c5e8-d887-4d53-bb2a-46e791a02ee6
```
Comment (alonkeyval) [inline tests/common/ui-tests/run_cypress_tests.sh:5]: also this key

File: tests/e2e/README.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+In addition to unit tests, Odigos has a suite of end-to-end tests that are run on every pull request.
+These tests are installing multiple microservices, instrument with Odigos, generate traffic, and validate the results.
+
+## Tools
+- [KinD](https://kind.sigs.k8s.io/) - Kubernetes in Docker is a tool for running local Kubernetes clusters using Docker container “nodes”.
+- [Chainsaw](https://kyverno.github.io/chainsaw/) - To orchestrate the different Kubernetes actions.
+- [Tempo](https://github.com/grafana/tempo) - Distributed tracing backend. Chosen due to its query language that allows for easy querying of traces.
+
+## Running e2e locally
+To run the end-to-end tests you need to have the following:
```
Comment (yodigos) [inline tests/e2e/README.md:11]: Add guide here to add .env

**Review by alonkeyval — APPROVED** [review]

**Pull request merged**